.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,256 @@
+        uses: actions/upload-artifact@v4
+        with:
+          name: CVODES.Ubuntu22
+          path: ./*.nupkg
+
+      - name: Publish to GitHub registry
+        # if it is a merge to default branch
+        if: github.event_name == 'push' && github.ref_name == github.event.repository.default_branch
+        run: dotnet nuget push ./*.nupkg --source https://nuget.pkg.github.com/${{github.repository_owner}}/index.json --api-key ${{ secrets.GITHUB_TOKEN }}
+
+  build-windows-x64:
+    runs-on: windows-latest
+    needs: get-version
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+
+      - name: define env variables
+        run: |
+          echo "APP_VERSION=${{needs.get-version.outputs.APP_VERSION}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+          echo "BUILD_ID=${{needs.get-version.outputs.BUILD_ID}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v2
+
+      - name: build debug
+        run: |
+          if not exist BuildCVODES_Windows mkdir BuildCVODES_Windows
+          echo Compiling for build type = Debug   
+          cmake -BBuildCVODES_Windows/Debug/x64/ -Hsrc/CVODES/ -DCMAKE_BUILD_TYPE=Debug -DEXAMPLES_ENABLE_C=OFF -DBUILD_SHARED_LIBS=OFF -DBUILD_STATIC_LIBS=ON -DENABLE_KLU=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DKLU_INCLUDE_DIR=BuildSuiteSparse/include/ -DKLU_LIBRARY_DIR=BuildSuiteSparse/lib64/ -DCMAKE_C_FLAGS_RELEASE="/O2 /Ob2 /DNDEBUG /Oi"
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+          msbuild BuildCVODES_Windows/Debug/x64/ALL_BUILD.vcxproj /property:Configuration=Debug
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+          if exist Dist\Windows\Debug\x64 rmdir /S /Q Dist\Windows\Debug\x64
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+          mkdir Dist\Windows\Debug\x64
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+        shell: cmd
+
+      - name: build release
+        run: |
+          if not exist BuildCVODES_Windows mkdir BuildCVODES_Windows
+          echo Compiling for build type = Release   
+          cmake -BBuildCVODES_Windows/Release/x64/ -Hsrc/CVODES/ -DCMAKE_BUILD_TYPE=Release -DEXAMPLES_ENABLE_C=OFF -DBUILD_SHARED_LIBS=OFF -DBUILD_STATIC_LIBS=ON -DENABLE_KLU=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DKLU_INCLUDE_DIR=BuildSuiteSparse/include/ -DKLU_LIBRARY_DIR=BuildSuiteSparse/lib64/ -DCMAKE_C_FLAGS_RELEASE="/O2 /Ob2 /DNDEBUG /Oi"
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+          msbuild BuildCVODES_Windows/Release/x64/ALL_BUILD.vcxproj /property:Configuration=Release
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+          if exist Dist\Windows\Release\x64 rmdir /S /Q Dist\Windows\Release\x64
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+          mkdir Dist\Windows\Release\x64
+          IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
+        shell: cmd
```

Comment:
use **build_Windows.bat** :slightly_smiling_face:

Reply:
ok, let's merge AS IS for now.
Created an issue: https://github.com/Open-Systems-Pharmacology/CVODES/issues/10